projects/js-packages/components/package.json

```
@@ -73,7 +73,10 @@
 		"extends @wordpress/browserslist-config"
 	],
 	"exports": {
-		".": "./index.ts",
+		".": {
+			"jetpack:src": "./index.ts",
+			"default": "./build/index.js"
```

What's the reason for this change to .js?

---

Now that we're producing built JS + .d.ts, it should be used for consumers outside the monorepo.

But now that you mention it, this should probably include
```suggestion
			"types": "./build/index.d.ts"
			"default": "./build/index.js"
```